pint/default_en.txt

```diff
@@ -174,7 +172,7 @@ week = 7 * day
 fortnight = 2 * week
 year = 365.25 * day = a = yr = julian_year
 month = year / 12
-decade = 10 * year
+# decade = 10 * year
```

Comment:
Add a note here indicating why this is commented. (We will have to make a decision about this and `B`)